static/lang/re-en.json

```diff
@@ -2906,15 +2906,15 @@
+                        "BaseTwo": "You can cast an amped <em>guidance</em> spell as a reaction triggered when your ally fails or critically fails an attack roll, Perception check, saving throw, or skill check, and the bonus from <em>guidance</em> would change the failure to a success or the critical failure to a normal failure. The bonus from <em>guidance</em> applies retroactively to their check.",
                         "Heightened": "The bonus from the spell increases to +2."
                     },
                     "HologramCage": "Force walls make the cage's barrier physical as well as visual. Each 10-foot-by-10-foot section of the cage's wall has AC 10, Hardness equal to double the spell's level, and HP equal to quadruple the spell's level. The spell is disrupted if any of its walls would pass through a creature when you cast it.",
                     "HomingBeacon": "Choose one creature hit by the spell (if the spell has a spell attack roll) or that fails its save against the spell (if the spell requires a save). A psychic beacon emits from the creature's location, making it easier to notice the creature as the mind and attention of onlookers is subtly drawn to it. For 1 minute, if the target would be @UUID[Compendium.pf2e.conditionitems.Item.zJxUflt9np0q4yML]{Invisible}, it's instead merely @UUID[Compendium.pf2e.conditionitems.Item.DmAIPqOBomZ7H95W]{Concealed}, and if it would be concealed, it's no longer concealed.",
                     "Ignition": {
-                        "Base": "You project pure heat that causes a target to combust. The initial damage changes to 1d10 fire damage plus 1 fire splash damage. When using amped ignition as a melee attack, increase the damage dice of the initial damage from d10s to d12s. You are not harmed by splash damage from amped ignition.",
-                        "Heightened": "Instead of using ignition’s normal heightened entry, the initial damage increases by 1d10 (1d12 for melee) and the splash damage increases by 1. The persistent fire damage on a critical hit increases by 1d4."
+                        "Base": "You project pure heat that causes a target to combust. The initial damage changes to 1d10 fire damage plus 1 fire splash damage. When using amped <em>ignition</em> as a melee attack, increase the damage dice of the initial damage from d10s to d12s. You are not harmed by splash damage from amped <em>ignition</em>.",
+                        "Heightened": "Instead of using <em>ignition's</em> normal heightened entry, the initial damage increases by 1d10 (1d12 for melee) and the splash damage increases by 1. The persistent fire damage on a critical hit increases by 1d4."
```

Comment:
```suggestion
                        "Heightened": "Instead of using <em>ignition</em>'s normal heightened entry, the initial damage increases by 1d10 (1d12 for melee) and the splash damage increases by 1. The persistent fire damage on a critical hit increases by 1d4."
```